_posts/2019-03-18-icing-rules.markdown

```diff
@@ -18,22 +18,22 @@ Zeus accordingly displayed his own cleverness by enchanting the boulder into rol
 Little is it known that after 3000 years King Sisyphus was able to reach the top of the mountain. There he found a Smirnoff Ice waiting for him placed by Zeus. Making Sisyphus the first in history to be iced and Zeus the almighty bro.
 
 ## Rules
+The rules are simple: If a person sees a Smirnoff Ice, he or she must get down on one knee and chug it, unless they happen to be carrying their own Smirnoff, in which case they can "ice block," or refract the punishment back onto the attacker. 
 
 ### Getting Iced
 1. The icee has to do it to themselves (i.e an action that reveals the ice. For example opening a door and see the ice).
 1. If an individual gets iced and no one is around to witness it they must live-stream it on Facebook.
 1. If you refuse to take the ice you will be shunned and booed for 2 days.
 
 ### Ice Block
-1. An individual can “ICE BLOCK” an ice if they have and ice in arms reach.
-1. This can stack and 1L are legal.
+1. An individual can _“ICE BLOCK”_ or _refract_ an ice if the individual has an ice in arms reach, and chooses to summon it.
+1. 1L are legal.
```

Comment:
```suggestion
1. 1L Ices are eligible to be blocked by another 1L Ice. If blocked by a smaller Ice, the icer must only drink twice that of the smaller ice, leaving the rest for the icee.
```

Reply:
Clarifies the blocking rules of liter ices with smaller ices

Reply:
This change is worthy of its own revision. @JackyChiu can you propose this?

Reply:
I will mark a separate proposal for 
```
If blocked by a smaller Ice, the icer must only drink twice that of the smaller ice, leaving the rest for the icee.
```

but I feel that this should be included to prevent ambiguous interpretation of what `1L are legal` means. 

```
1L Ices are eligible to be blocked by another 1L Ice.
```